build-support/bin/cache_comparison.py

```diff
@@ -11,12 +11,16 @@
 import sys
 from time import time
 
+from pants.util.strutil import softwrap
```

Comment:
This means that you cannot run the script directly anymore. I don't think it's worth it